### ISSUE TITLE
proposal logo for steemit IDCard

### ISSUE THREAD

**Issue by ivannewgate89** [iamank1t/Steemit-IDCard#1]
hi iamank1k, i have idea for youre logo, can i join?

**Comment by ivannewgate89** [comment]
so, in this logo is not allowed to use the steemit logo, but what is not problematic on the color. What do you think?

**Comment by ivannewgate89** [comment]
sir, i think you can change name logo become steem idcard, and i have idea for this logo

**Comment by iamank1t** [comment]
yeah sure... i will be glad to hear any type of suggestion from your side.
